src/javascript/app_2/App/Components/Form/input_field.jsx

```diff
@@ -41,14 +41,18 @@ const InputField = ({
             // This regex check whether there is any zero at the end of fractional part or not.
             const has_zero_at_end = new RegExp(`^${signed_regex}(\\d+)?\\.(\\d+)?[0]+$`)
                 .test(e.target.value);
+            
+            const scientific_notation = /e/.test(`${+e.target.value}`);
```

Comment:
Shall we start the name of variables that store boolean value with `is_`, `has_`, ...?